src/loaders/TiffLoader.ts

```diff
@@ -184,6 +184,15 @@ class TiffLoader extends ThreadableVolumeLoader {
 
       numMultiscaleLevels: 1,
       multiscaleLevel: 0,
+      multiscaleLevelDims: [
+        {
+          shape: [dims.sizet, dims.sizec, dims.sizez, dims.sizey, dims.sizex],
+          spacing: [1, 1, dims.pixelsizez, dims.pixelsizey, dims.pixelsizex],
+          spaceUnit: dims.unit || "",
+          timeUnit: "",
+          dataType: dims.pixeltype || "uint8",
+        },
+      ],
```

Comment:
Kind of odd to see this structure being explicitly made inline in all these loaders, when `IVolumeLoader` requires a different method explicitly for getting them. I get why you did it this way - calling a loader's `loadDims` could re-fire whatever `async` operation grabbed that metadata in the first place - but if `createImageInfo` is going to effectively produce a superset of what `loadDims` gives you, it makes me wonder if we still need `loadDims` in the first place.

More generally, I think there's some growing unresolved confusion around what our different image metadata types (`VolumeDims` vs `ImageInfo` vs `LoadSpec`) are for, though I'm fine leaving it unresolved for this PR.

Reply:
I agree it's a good candidate for a further change after this